static/src/javascripts/projects/common/modules/identity/api.js

```diff
@@ -163,13 +164,19 @@ export const getUserEmailSignUps = (): Promise<any> => {
 };
 
 export const sendValidationEmail = (): any => {
+    const returnEndpoint = '/email-prefs';
```

Comment:
would this be better named `defaultReturnEndpoint ` or `fallbackReturnEndpoint` or something along those lines?